### DISCUSSION TITLE
[6 giugno 2024] Complessità

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/automi-calcolabilita-complessita#30] [Complessità]
![image](https://github.com/sapienzastudentsnetwork/acc2324/assets/69389461/7f4aaf86-f3a5-45ff-a499-f7b330ed8950)

**Comment by Exyss** [comment]
Per il primo punto, ripropongo la soluzione presente sui miei appunti:

![image](https://github.com/user-attachments/assets/d6f944fc-d860-4d53-940c-dfbcd160d063)


----

La dimostrazione richiesta nel secondo punto è presente sul libro di testo

**Comment by daromacarlo** [comment]
Funziona?
<img width="964" height="801" alt="immagine" src="https://github.com/user-attachments/assets/3f7b4d76-17d8-4200-8e6d-a91388745606" />
